New CN value in connector-service (#2757)
kyma-project/kyma: components/connector-service/README.md

- **caSecretName** - Name of the secret which contains the root Certificate Authority (CA). The default value is `nginx-auth-ca`.
- **requestLogging** - Flag for logging incoming requests. It is set to `False` by default.
- **connectorServiceHost** - Host under which this service is accessible. It is used for generating the URL. The default host is `cert-service.wormhole.cluster.kyma.cx`.

> - **gatewayHost** - Host at which gateway service is accessible The default value is `gateway.wormhole.cluster.kyma.cx`.

Reviewer: And generally, we have a rule according to which we should write:
```
-  **gatewayHost** is the host...
```

so I would basically remove `-` everywhere and add `verb + the {noun}` everywhere here :)

- **certificateProtectedHost** - Host secured with client certificate, used for certificate renewal. The default host is `gateway.wormhole.cluster.kyma.cx`.
- **appsInfoURL** - URL at which management information for applications is available. If not provided, it bases on `connectorServiceHost`.

> - **runtimesInfoURL** - URL at which management information for runtimes is available. If not provided, it bases on `connectorServiceHost`.

Reviewer: ```suggestion
- **runtimesInfoURL** - URL at which the management information for runtimes is available. If not provided, it bases on `connectorServiceHost`.
```

> - **certificateValidityTime** - Validity time of certificates issued by this service. The default value is 90 days.

Reviewer: ```suggestion
- **certificateValidityTime** - Time until which the certificates that the service issues are valid. The default value is 90 days.
```
